project1/templates/book.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            BOOK PAGE
+        </title>
+        <style>
+            h1{
+                text-align: center;
+                color: black;
+            }
+            tr th{
+                color:blue;
+            }
+            tr td{
+                color:blueviolet
+            }
+            .btn {
+            background-color: rgb(71, 165, 223);
+            color: white;
+            padding: 16px 20px;
+            margin : 8px 0;
+            margin-left: auto;
+            border: none;
+            cursor: pointer;
+            width: 50%;
+            opacity: 1;
+            
+        }
+        body{
+            text-align: center;
+        }
+        table{
+            margin-left: auto;
+            margin-right: auto;
+            font-size: 20px;
+        }
+        </style>
+    </head>
+    <body style="background-color: antiquewhite;">
+        <h1>
+            Welcome to the details of the book you are searching for...
+        </h1>
+        <table style="width:30%;">
+            {% for each in book_details %}
```

Comment:
@hemanthrachala I think so in book page we will have only one book details to display. And book details page also include reviews in its page, which was done by @shaiksaajid11 . What did you do extra here???